lib/plugins/aws/deploy/compile/events/schedule/index.js

```diff
@@ -38,9 +40,19 @@ class AwsCompileScheduledEvents {
               }
               ScheduleExpression = event.schedule.rate;
               State = event.schedule.enabled ? 'ENABLED' : 'DISABLED';
+              Input = event.schedule.input || '';
+              InputPath = event.schedule.inputPath || '';
```

Comment:
Well just testing it now and the error is triggered lately by cloud formation:

```
An error occurred while provisioning your stack: HelloEventsRuleSchedule1
     - Both Input and InputPath are provided for target helloSchedule..
```

I'd say that's good enough! :)